Library/Homebrew/test/rubocops/lines_cop_spec.rb

```diff
@@ -251,7 +251,7 @@ class Foo < Formula
       expected_offenses.zip(cop.offenses).each do |expected, actual|
         expect_offense(expected, actual)
       end
-    end
+    end if OS.mac?
```

Comment:
Try changing line 229 to
```ruby
it "with OS.linux? check", :needs_macos do
```
I'm not certain that's the correct syntax, but give it a go.